spock-specs/src/test/groovy/org/spockframework/smoke/extension/PendingFeatureExtensionSpec.groovy

```diff
@@ -39,6 +39,29 @@ def bar() {
     }
   }
 
+  def "@PendingFeature marks failing feature as skipped even if applied twice"() {
+    when:
+    def result = runner.runSpecBody """
+@PendingFeature
```

Comment:
While not "wrong", it is redundant, we should check for redundant annotations, as the user probably intended something different.

Reply:
The question is, how much logic to stuff in for those redundancy checks and how much to trust the user.
You can also prevent multiple `@Issue` annotations with the same value or.
Or you could even prevent `@Issue(['1', '2'], ['2', '3']) as the `'2'` is redundant.
Same for two `@PendingFeature` annotations with the same expected exception in their lists.